roles/ceph-mon/defaults/main.yml

```diff
@@ -10,6 +10,7 @@ fetch_directory: fetch/
 mon_group_name: mons
 
 # ACTIVATE BOTH FSID AND MONITOR_SECRET VARIABLES FOR NON-VAGRANT DEPLOYMENT
+# Comment out fsid when using mon_containerized_deployment_with_kv

```

Comment:
This one is tricky, how do you make sure the fsid in the ceph.conf generated will match with the one generated by ceph-docker during the init monitor bootstrap?
We can add an option in ceph-docker to specify a FSID and then re-use the same here.